contract-examples/tsconfig.json

```diff
@@ -1,5 +1,19 @@
 {
   "compilerOptions": {
-    "resolveJsonModule": true
-  }
+    "outDir": "./dist/",
+    "baseUrl": ".",
+    "module": "commonjs",
+    "target": "es2020",
+    "lib": ["dom", "es2016"],
```

Comment:
I don't believe you need the `dom` library

Reply:
right 👍 deleted.